WordPress/Classes/ViewRelated/Site Creation/Site Name/SiteNameView.swift

```diff
@@ -136,9 +142,12 @@ private extension SiteNameView {
 
     /// Highlghts the site name in blue
     func setupTitleColors() {
-        let fullTitle = String(format: TextContent.title, siteName)
+        if !siteVerticalName.isEmpty {
+            siteVerticalName = " " + siteVerticalName
```

Comment:
Oh, I think this may be related to hitting "Skip".

Reply:
it was, but nonetheless thanks for pointing this out, because that code can certainly be improved and we don't need this hack, just add another check on the `guard` that already checks the range of the vertical name, and that's it 🤦